### PR TITLE
CO2-H2O VLE Property Package Tutorial

### PR DESCRIPTION
**Pull request by anujad95**
## Fixes # .
NA
## Proposed changes:
- Adds a tutorial for developing a CO2-H2O property package based on the existing generic property framework in IDAES, and implementing it for a flash unit model simulation

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.txt file at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

### CODE REVIEW

**Comment by ksbeattie** [comment]
@jghouse88, can you take a look?

**Comment by jghouse88** [comment]
@andrewlee94 I have reviewed this and made some changes to the text and added a testing cell. I would like you to go over the description especially the sections that describe the idaes components required for building the config dict.

**Review by jghouse88 — APPROVED** [review]
LGTM

File: src/Examples/Advanced/CustomProperties/ideal_h2o_co2_vle_prop_pack_testing.ipynb [hunk 1]
```diff
@@ -0,0 +1,689 @@
...
+    "[2] Perry's Chemical Engineers' Handbook 7th Ed."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "configuration = {\n",
```
Comment (andrewlee94) [inline src/Examples/Advanced/CustomProperties/ideal_h2o_co2_vle_prop_pack_testing.ipynb:316]: I think we need more description here. Given that this is basically a tutorial, I think the first part should provide a general description of the overall tool and what each part does and why you want to set it. The second part (i.e. here) should then show **how** to set each part for a given application with some justification/discussion of why/how each value was chosen.

**Review by andrewlee94 — COMMENTED** [review]
I think we need to think a little bit more about the audience for this example and provide more description. This is basically the first introduction to the generic property framework, so I think the first section needs to explain what the generic framework is and what it does - i.e. part 1 should be more what and why rather than how. Part 2 can then be the how, with an example of a specific system (CO2-H2O), and Part 3 then being a demonstration of how to put all of this to use in a flowsheet.

**Comment by ksbeattie** [comment]
@anujad95, any update on this?

**Comment by ksbeattie** [comment]
@anujad95 what about this one?

**Comment by ksbeattie** [comment]
@anujad95 is this still possible for the Aug release?

**Comment by anujad95** [comment]
@ksbeattie I will try to get through this as soon as possible.

**Comment by ksbeattie** [comment]
@anujad95, moving this off of the Nov release to the Feb release.  Please let me know if that shouldn't be done.

**Comment by ksbeattie** [comment]
@anujad95, I'm going to close this.  Please re-open or re-create if/when you get back to it.

**Pull request closed without merging**